src/plugins/data_source_management/public/components/data_source_view/data_source_view.tsx

```diff
@@ -78,67 +85,44 @@ export class DataSourceView extends React.Component<DataSourceViewProps, DataSou
     this.setState({ ...this.state, isPopoverOpen: false });
   }
 
-  getPanels() {
-    let items: MenuPanelItem[] = [];
-    if (this.state.selectedOption) {
-      items = this.state.selectedOption.map((option) => {
-        return {
-          name: option.label,
-          disabled: true,
-        };
-      });
-    }
-
-    const panels = [
-      {
-        id: 0,
-        title: 'Selected data source',
-        items,
-      },
-    ];
-
-    return { panels };
-  }
-
   render() {
-    const { panels } = this.getPanels();
-
-    const button = (
-      <EuiButtonIcon
-        iconType="iInCircle"
-        display="empty"
-        aria-label="Next"
-        onClick={this.onClick.bind(this)}
-      />
-    );
+    const label =
+      this.state.selectedOption && this.state.selectedOption.length > 0
+        ? this.state.selectedOption[0].label
+        : '';
+    const id =
+      this.state.selectedOption && this.state.selectedOption.length > 0
+        ? this.state.selectedOption[0].id
+        : '';
     return (
-      <>
-        <EuiButtonEmpty
-          className="euiHeaderLink"
-          data-test-subj="dataSourceViewContextMenuHeaderLink"
-          aria-label={i18n.translate('dataSourceView.dataSourceOptionsButtonAriaLabel', {
-            defaultMessage: 'dataSourceViewMenuButton',
-          })}
-          iconType="database"
-          iconSide="left"
-          size="s"
-          disabled={true}
-        >
-          {this.state.selectedOption && this.state.selectedOption.length > 0
-            ? this.state.selectedOption[0].label
-            : ''}
-        </EuiButtonEmpty>
-        <EuiPopover
-          id={'dataSourceViewContextMenuPopover'}
-          button={button}
-          isOpen={this.state.isPopoverOpen}
-          closePopover={this.closePopover.bind(this)}
-          panelPaddingSize="none"
-          anchorPosition="downLeft"
+      <div>
+        <EuiToolTip
+          data-test-subj="dataSourceViewrToolTip"
+          title={<div style={{ width: '300px' }}>SELECTED DATA SOURCE</div>}
```

Comment:
can we move style into a css file? Does the width break mobile view?

Reply:
Would do